configure.ac

```
@@ -202,6 +204,151 @@ AS_CASE([$with_gc],
     )
 AC_MSG_RESULT([$with_gc])
 
+dnl
+dnl User setting: disable guards (race protection)
+dnl
+
+AC_ARG_ENABLE([guards],
+    [AS_HELP_STRING([--enable-guards],
+        [disable race protection in HPC-GAP])],
```

Hmm, why don't the name of the option and the description not match? I'd expect it either to be called `--enable-race-protection`, or else a description which actually matches the option.
Also, the option says "--enable-guard", but the description talks about **disable** ?

 How about:
```suggestion
        [disable guards for race protection in HPC-GAP])],
```
or if the verb was wrong, then
```suggestion
        [enable guards for race protection in HPC-GAP])],
```

---

Oops, I flipped the semantics, but not the description. Fixed.